src/GuiBase/Viewer/TrackballCamera.cpp

```diff
@@ -13,8 +14,28 @@
 #include <iostream>
 
 namespace Ra {
+namespace Gui {
 using Core::Math::Pi;
 
+#define KMA_VALUE( XX ) Gui::KeyMappingManager::KeyMappingAction Gui::TrackballCamera::XX;
+KeyMappingCamera;
+#undef KMA_VALUE
+
+void Gui::TrackballCamera::registerKeyMapping() {
+
+    m_keyMappingContext = Gui::KeyMappingManager::getInstance()->getContext( "CameraContext" );
+    if ( m_keyMappingContext.isInvalid() )
+    {
+        LOG( Ra::Core::Utils::logINFO )
+            << "ViewerContext not defined (maybe the configuration file do not contains it";
+    }
+
+#define KMA_VALUE( XX ) \
+    XX = Gui::KeyMappingManager::getInstance()->getActionIndex( m_keyMappingContext, #XX );
+    KeyMappingCamera
+#undef KMA_VALUE
```

Comment:
In case the context is invalid we should return and not try to access it, otherwise this will crash.
Also, please update the log message from `ViewerContext` to `CameraContext`